integration/test_resources/help/agent_aws-install_ec2ic

```diff
@@ -0,0 +1,52 @@
+This command installs the agent on all EC2 instances in an AWS account using EC2InstanceConnect.
+
+To filter by one or more regions:
+
+    lacework agent install ec2ic --include_regions us-west-2,us-east-2
```

Comment:
Please change all instances of "lacework agent install ec2ic" to "lacework agent aws-install ec2ic" in this file.